django_socio_grpc/tests/test_async_model_service.py

```diff
@@ -183,3 +186,45 @@ async def test_async_retrieve(self):
         response = await grpc_stub.Retrieve(request=request)
 
         self.assertEqual(response.proto_slug_related_field, str(self.foreign.uuid))
+
+
+@override_settings(GRPC_FRAMEWORK={"GRPC_ASYNC": True})
+class TestSimpleRelatedFieldModelService(TestCase):
+    def setUp(self):
+        self.fake_grpc = FakeFullAIOGRPC(
+            add_SimpleRelatedFieldModelControllerServicer_to_server,
+            SimpleRelatedFieldModelService.as_servicer(),
+        )
+
+    def tearDown(self):
+        self.fake_grpc.close()
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.item1 = RelatedFieldModel.objects.create()
+        cls.foreign = ForeignModel.objects.create(name="foreign")
+        cls.item2 = RelatedFieldModel.objects.create(foreign=cls.foreign)
+
+    async def test_async_retrieve(self):
+        grpc_stub = self.fake_grpc.get_fake_stub(SimpleRelatedFieldModelControllerStub)
+
+        request = fakeapp_pb2.SimpleRelatedFieldModelRetrieveRequest(uuid=str(self.item1.uuid))
+        response = await grpc_stub.Retrieve(request=request)
+
+        self.assertEqual(response.foreign, "")
+
+        request = fakeapp_pb2.SimpleRelatedFieldModelRetrieveRequest(uuid=str(self.item2.uuid))
+        response = await grpc_stub.Retrieve(request=request)
+
+        self.assertEqual(response.foreign, str(self.foreign.uuid))
+
+    async def test_async_create(self):
```

Comment:
miss comment to explain that it test the _ConvertScalarFieldValue functionnality